lib/rest-api.php

```diff
@@ -18,3 +18,88 @@ function gutenberg_register_global_styles_endpoints() {
 	$global_styles_controller->register_routes();
 }
 add_action( 'rest_api_init', 'gutenberg_register_global_styles_endpoints' );
+
+/**
+ * Registers additional fields for search result rest api.
+ *
+ * @access private
```

Comment:
I might be wrong here but I think these changes need to go in `lib/compat/wordpress-6.6/rest-api.php`. I think it's fine to create the folder too